mmv1/products/vertexai/Index.yaml

```diff
@@ -100,6 +100,7 @@ properties:
     properties:
       - !ruby/object:Api::Type::String
         name: 'contentsDeltaUri'
+        required: true
```

Comment:
This is a breaking change and can usually only be added within a major release (of which we have a very short window for this to get into)

Additionally, I cannot find it written that this field is required in all scenarios. The wording in https://cloud.google.com/vertex-ai/docs/vector-search/configuring-indexes makes it seem like it is not strictly required, where other required fields are explicitly labeled so. What do you think?

Reply:
> This is a breaking change and can usually only be added within a major release

Hello @c2thorn, this change looks to correspond to the second example. So I believed we can release this in a minor release.

https://github.com/GoogleCloudPlatform/magic-modules/blob/aef0da09e868754d947f4c4c932af5eae7ef8e5c/docs/content/develop/make-a-breaking-change.md?plain=1#L41-L54

> Additionally, I cannot find it written that this field is required in all scenarios.

That's true. I couldn't find any description either. I also want to know if it's optional or required. Can you reach out to the product team? Or should I reach out to the team by creating a support ticket or asking a technical account manager?

Reply:
I'll reach out to the product team

Reply:
It's required on Create but not Update, which is maybe why the wording is not concrete. I think moving forward with marking it as required overall is the best choice.

> Hello @c2thorn, this change looks to correspond to the second example. So I believed we can release this in a minor release.

Thanks for pointing this out, that looks to be correct. Let's move forward with it.

Reply:
> It's required on Create but not Update
I see. Thank you for letting me know!